R/tm_g_distribution.R

```diff
@@ -1,33 +1,33 @@
-#' Distribution Module
-#' @md
+#' Distribution analysis module
 #'
-#' @details
-#' Module to analyze and explore univariate variable distribution
+#' Module is designed to explore the distribution of a single variable within a given dataset.
+#' It offers several tools, such as histograms, Q-Q plots, and various statistical tests to
+#' visually and statistically analyze the variable's distribution.
 #'
 #' @inheritParams teal::module
 #' @inheritParams teal.widgets::standard_layout
 #' @inheritParams shared_params
 #'
 #' @param dist_var (`data_extract_spec` or `list` of multiple `data_extract_spec`)
-#'  Variable to consider for the distribution analysis.
+#' Variable(s) for which the distribution  to be analyzed.
 #' @param strata_var (`data_extract_spec` or `list` of multiple `data_extract_spec`)
-#'   Categorical variable to split the selected distribution variable on.
-#' @param group_var  optional, (`data_extract_spec` or `list` of multiple `data_extract_spec`)
-#'   Which data columns to use for faceting rows.
-#' @param freq optional, (`logical`) Whether to display frequency (`TRUE`) or density (`FALSE`).
-#'   Defaults to density (`FALSE`).
-#' @param bins optional, (`integer(1)` or `integer(3)`) If scalar then the histogram bins will have a fixed size.
-#'   If a slider should be presented to adjust the number of histogram bins dynamically then it can be a
-#'   vector of length three with `c(value, min, max)`.
-#'   Defaults to `c(30L, 1L, 100L)`.
+#' Categorical variable used to split the distribution analysis.
+#' @param group_var (`data_extract_spec` or `list` of multiple `data_extract_spec`)
+#' Variable used for faceting plot into multiple panels.
+#' @param freq (`logical`, optional) Whether to display frequency (`TRUE`) or density (`FALSE`).
+#' Defaults to density (`FALSE`).
+#' @param bins (`integer(1)` or `integer(3)`, optional)  Determines the number of bins for the histogram.
+#' - If vector of `length == 1` then the histogram bins will have a fixed size.
+#' - while vector of `value`, `min`, and `max` allows histogram bins dynamic adjustment.
```

Comment:
`length` as a variable is not declared so using it might cause confustion.
I liked the previous version when it was clear that the values passed should be `c(value, min, max)`
As a document reader, it makes it easier for me to write code after seeing that instead of reading the meaning and coming up with `c(...)`.

```suggestion
#' - When the length of `bins` is one: The histogram bins will have a fixed size based on the `bins` provided.
#' - When the length of `bins` is three: The histogram bins are dynamically adjusted based on `c(value, min, max)`.
```

Reply:
I've updated the c(..) usage in all the pull requests due to a review comment that I can't locate now. To maintain consistency, let's stick with this change.

I will incorporate rest of the change.

Reply:
Got it. Then I'll resolve and we can merge this one in.